src/main/java/com/amazon/opendistroforelasticsearch/security/dlic/rest/api/AbstractApiAction.java

```diff
@@ -346,7 +346,7 @@ public ConfigUpdatingActionListener(Client client, ActionListener<Response> dele
 		@Override
 		public void onResponse(Response response) {
 
-			final ConfigUpdateRequest cur = new ConfigUpdateRequest(new String[] { "config", "roles", "rolesmapping", "internalusers", "actiongroups" });
+			final ConfigUpdateRequest cur = new ConfigUpdateRequest(ConfigConstants.ALL_CONFIG_NAMES.toArray(new String[0]));
```

Comment:
nit: Is it possible to make another constant variable for this? 
`ConfigConstants.ALL_CONFIG_NAMES.toArray(new String[0])` as it seems to be used quite a lot.